spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/RabbitConnectionFactoryBean.java

```diff
@@ -121,6 +123,27 @@
 
 	private volatile SecureRandom secureRandom;
 
+	private boolean skipServerCertificateValidation;
+
+
+	/**
+	 *
+	 * @return whether or not Server Side certificate has to be validated or not
+     */
+	public boolean isSkipServerCertificateValidation() {
+		return this.skipServerCertificateValidation;
+	}
+
+	/**
+	 * whether or not Server Side certificate has to be validated or not
+	 * this would be used if useSSL is set to true and should only be used on dev or Qa regions
+	 * skipServerCertificateValidation should never be set to true in production
+	 *
```

Comment:
Need `@param`:
```
RabbitConnectionFactoryBean.java:143: warning: no @param for skipServerCertificateValidation

	public void setSkipServerCertificateValidation(boolean skipServerCertificateValidation) {
```
Also `@since 1.6.6`